keyboards/lazydesigners/the60/readme.md

```diff
@@ -0,0 +1,18 @@
+# THE60
+
+![THE60](https://cdn.discordapp.com/attachments/536190211240099840/549506683064287243/image0.jpg)
+
+A 60% custom PCB and case designed and produced by [LazyDesigners](http://lazydesigners.cn).  
+The PCB allows one to test a 40% layout in a 60% case.
+
+Keyboard Maintainer: QMK Community  
+Hardware Supported: THE60 PCB  
+Hardware Availability: Check for GBs on [Geekhack](https://geekhack.org) and on [LazyDesigner's homepage.](http://lazydesigners.cn)  
```

Comment:
can you put direct links to the GBs please?

Reply:
There hasn't been any. To my best knowledge, a GB will be in the coming months.

Reply:
Alright we can update this readme with more info when the time comes.